.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
let's use the global frame labs prettier config instead

Reply:
I get 404 trying to install - can we publish this config?